source/nodejs/adaptivecards-designer/src/card-designer.ts

```diff
@@ -431,20 +430,20 @@ export class CardDesigner extends Designer.DesignContext {
         this.recreateDesignerSurface();
 
         if (this._deviceEmulationChoicePicker) {
-            this._deviceEmulationChoicePicker.isEnabled = !!this.hostContainer.enableDeviceEmulation;
+            this._deviceEmulationChoicePicker.isHidden = !(!!this.hostContainer.enableDeviceEmulation);
             this.activeDeviceEmulationChanged();
         }
 
         if (this._containerSizeChoicePicker) {
-            this._containerSizeChoicePicker.isEnabled = !!this.hostContainer.supportsMultipleSizes;
+            this._containerSizeChoicePicker.isHidden = !(!!this.hostContainer.supportsMultipleSizes);
 
             // Update the host parameter data with the value of the choice picker
             if (this._containerSizeChoicePicker.isEnabled && this._sampleHostData) {
                 this.updateHostDataSizeProperty();
 
                 // If the container properties do not align with the choice picker, update the property and recreate the designer surface
-                if (this._containerSizeChoicePicker.value !== this._hostContainer.containerSize) {
-                    this._hostContainer.containerSize = this._containerSizeChoicePicker.value as ContainerSize;
+                if (this._containerSizeChoicePicker.value !== (this._hostContainer as BerlinContainer).containerSize) {
+                    (this._hostContainer as BerlinContainer).containerSize = this._containerSizeChoicePicker.value as ContainerSize;
```

Comment:
There would be an issue if `this._hostContainer` isn't a `BerlinContainer`, so I'll add an `instanceof` check.

We shouldn't hit that scenario because `this._containerSizeChoicePicker.isEnabled` should only set if hostContainer is a BerlinContainer, but good to have the check in case we change the implementation.